Makefile

```diff
@@ -69,3 +70,19 @@ flake: ## 🧹 Run flake8
 updatekv: ## 🔄 Update keyvault secrets
 	$(call target_title, "updating keyvault secrets")
 	python3 env_to_keyvault.py
+include .env
+export
+
+deploy:
+	az deployment sub create \
+		--location $(LOCATION) \
+		--template-file $(MAIN_BICEP_FILE)
+
+deploy_with_isolated_network:
```

Comment:
Do you validate that the three env variables have been set?
e.g. do you check that `VIRTUAL_NETWORK_ADDRESS_SPACE` has a value, and isn't an empty string?

Reply:
Template validation fails if it has invalid addresses.
But yeah think best practice will be to check it with if statement and throw error rather than replying on template validation.
I will update this to include the param value check. :)

Reply:
Code now check the three address space inputs are not empty.